package.json

```diff
@@ -362,43 +362,43 @@
     "graceful-fs": "^4.1.2",
     "http-proxy": "^1.13.0",
     "isbinaryfile": "^3.0.0",
-    "lodash": "^4.17.4",
-    "log4js": "^2.3.9",
-    "mime": "^1.3.4",
+    "lodash": "^4.17.10",
+    "log4js": "^2.6.0",
+    "mime": "^2.3.1",
     "minimatch": "^3.0.2",
     "optimist": "^0.6.1",
     "qjobs": "^1.1.4",
     "range-parser": "^1.2.0",
     "rimraf": "^2.6.0",
     "safe-buffer": "^5.0.1",
-    "socket.io": "2.0.4",
-    "source-map": "^0.6.1",
+    "socket.io": "2.1.0",
+    "source-map": "^0.7.2",
```

Comment:
Please note that the `SourceMapConsumer` constructor [became `async`](https://github.com/mozilla/source-map/blob/master/CHANGELOG.md#070) in `source-map` 0.7.0. We should change the code where it's used (`lib/reporter.js`) accordingly.

Reply:
Plus the Travis complains:
 `source-map@0.7.2: The engine "node" is incompatible with this module. Expected version ">= 8".`

Reply:
right, there was something.. undid that upgrade